test/functional/apps/discover/_context_encoded_url_params.ts

```diff
@@ -24,15 +24,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await security.testUser.setRoles(['kibana_admin', 'context_encoded_param']);
       await PageObjects.common.navigateToApp('settings');
       await es.transport.request({
-        path: `/context-encoded-param/_doc/${customDocIdParam}`,
+        path: `/context_encoded_param/_doc/${customDocIdParam}`,
```

Comment:
I think this change would break other this test in cloud? 
![Uploading Bildschirmfoto 2022-06-27 um 11.53.02.png…]()

Reply:
Thx for catch